recipes/libs/libcap-ng.yaml

```diff
@@ -0,0 +1,21 @@
+inherit: [autotools]
+
+metaEnvironment:
+    PKG_VERSION: "0.8.5"
+    PKG_LICENSE: "GPL-2.0+ (programs), LGPL-2.1+ (library)"
```

Comment:
Does this imply that it's not just a library but also has some executables? Then we should separate them into distinct packages. The `PKG_LICENSE` can then be attached to each part individually.

Reply:
done